### ISSUE TITLE
Able to install the same toolchain twice

### ISSUE THREAD

**Issue by drhodes** [leanprover/elan#126]
This is kind of weird, we can install two of the same toolchain:

```bash
$ elan toolchain install v4.7.0
$ elan toolchain install 4.7.0
```

elan will list them as two distinct installs

```
$ elan toolchain list
v4.7.0
4.7.0
```

**Comment by Kha** [comment]
Fixed in #106

**Comment by drhodes** [comment]
That was fast, thank you. Sorry, I guess I didn't check if this was a dupe.
